### DISCUSSION TITLE
Problem with ping

### DISCUSSION THREAD

**Discussion by onLaait** [Minestom/Minestom#1703] [Help] [answered]
My ping is displayed as over 60 even on localhost.
Is there a solution to this?

**Comment by KrystilizeNevaDies** [comment, marked as answer]
There is no accurate way to measure ping unfortunately (except for serverlist). This is because of server - client desync.

Minestom processes packets every 50 ms, same as the vanilla client.
This means that there will be between 0-100 extra ms added on the each ping. The only real way around this is modding the vanilla client heavily and forking minestom.

**Reply by KrystilizeNevaDies** [reply]
I should note that the 0-100 extra ms of ping has some technicalities and is a rough extimate.
